source/base/ThreadPool.ooc

```diff
@@ -54,8 +55,8 @@ _ActionTask: class extends _Task {
 _ResultTask: class <T> extends _Task {
 	_result: Cell<T>
 	_action: Func -> T
-	init: func (=_action, =_mutex)
-	run: func (=_mutex) {
```

Comment:
Wait, I remember now. It wasn't as big a problem as I thought it was. Still, this new solution is much better.